rs/execution_environment/src/metrics.rs

```diff
@@ -268,8 +271,27 @@ impl QueryHandlerMetrics {
                 "The total number of transient errors accumulated \
                         during the query execution",
             ),
+            subnet_query_messages: metrics_registry.histogram_vec(
+                "execution_subnet_query_message_duration_seconds",
```

Comment:
Could we maybe use the same name? Would it work? Or would we have to set all labels which wouldn't make sense for the query metric?